### ISSUE TITLE
Import CSV etc from other form plugins into BlockForm.

### ISSUE THREAD

**Issue by paaljoachim** [Pyrometric/BlockForm#22]
User case.

I renewed/changed the theme for a site and basically just redid most of the pages for this small site.
It uses Gravity Forms. The new site uses Caldera Forms. I am looking into exporting the CSV from Gravity Forms and importing it into Caldera Forms and when ready hopefully this year before Caldera Forms retires import these into BlockForm. I am hoping it will work. I have yet to try it out.

Bottom line is: Being able to export forms and entries from various forms into BlockForm.

------

I end up doing it the hard way. Manually copying every entry into a document for the customer. As all the entries would be erased when I added the new dev site on top of the online site. 
(There are very very likely much easier ways of doing this but today well I did it the hard way.)
